Tests/DuplicatedSDKTest/DuplicatedSDKTest/DuplicatedSDKTestApp.swift

```diff
@@ -0,0 +1,50 @@
+import ObjectiveC.runtime
+import Sentry
+import SwiftUI
+
+@main
+struct DuplicatedSDKTestApp: App {
```

Comment:
`m`: A quick comment explaining why we have this sample app would be great cause it's not that obvious.